### PR TITLE
Add language mapping and update client

### PR DESCRIPTION
**Pull request by frombetelgeuse**

### CODE REVIEW

File: src/main/java/com/crowdin/util/LanguageMapping.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+     *         "uk": "ua"
+     *     },
+     *     "name": {
+     *         "uk": "Ukrainian"
+     *     }
+     * }
+     * @param fileLanguageMapping langauge mapping from configuration file. May be null/empty
+     * @return immutable LanguageMapping object
+     */
+    public static LanguageMapping fromConfigFileLanguageMapping(Map<String, Map<String, String>> fileLanguageMapping) {
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/util/LanguageMapping.java:69]: Seems like this method is unused

File: src/main/java/com/crowdin/client/CrowdinProjectCacheProvider.java [hunk 2]
```diff
@@ -52,14 +119,17 @@ public synchronized static CrowdinProjectCache getInstance(Crowdin crowdin, Stri
             return crowdinProjectCache;
         }
         Branch branch = crowdinProjectCache.getBranches().get(branchName);
-        if (!crowdinProjectCache.getFiles().containsKey(branch)
+        System.out.println("!!!!!!!!!!!!!!!!!!!!!!! here");
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/client/CrowdinProjectCacheProvider.java:122]: ?

File: src/main/java/com/crowdin/client/CrowdinProjectCacheProvider.java [hunk 3]
```diff
@@ -52,14 +119,17 @@ public synchronized static CrowdinProjectCache getInstance(Crowdin crowdin, Stri
...
+        if (!crowdinProjectCache.getFileInfos().containsKey(branch)
                 || !crowdinProjectCache.getDirs().containsKey(branch)
                 || outdatedBranches.contains(branchName)
                 || update) {
             Long branchId = (branch != null) ? branch.getId() : null;
-            List<com.crowdin.client.sourcefiles.model.File> files = crowdin.getFiles(branchId);
+            List<com.crowdin.client.sourcefiles.model.FileInfo> files = crowdin.getFiles(branchId);
             Map<Long, Directory> dirs = crowdin.getDirectories(branchId);
-            crowdinProjectCache.getFiles().put(branch, CrowdinFileUtil.buildFilePaths(files, dirs));
+            System.out.println("!!!!!!!!!!!!!!!!! updating files: " + files.size());
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/client/CrowdinProjectCacheProvider.java:130]: Do we need this?

File: src/main/java/com/crowdin/client/CrowdinProjectCacheProvider.java [hunk 4]
```diff
@@ -52,14 +119,17 @@ public synchronized static CrowdinProjectCache getInstance(Crowdin crowdin, Stri
...
+            crowdinProjectCache.getFileInfos().put(branch, CrowdinFileUtil.buildFilePaths(files, dirs));
+            System.out.println("crowdinProjectCache.getFileInfos() = " + crowdinProjectCache.getFileInfos());
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/client/CrowdinProjectCacheProvider.java:132]: Do we need this?

File: src/main/java/com/crowdin/util/LanguageMapping.java (hunk 1 above)
Comment (frombetelgeuse) [reply src/main/java/com/crowdin/util/LanguageMapping.java:69]: Yes, you're right

File: src/main/java/com/crowdin/client/CrowdinProjectCacheProvider.java (hunk 2 above)
Comment (frombetelgeuse) [reply src/main/java/com/crowdin/client/CrowdinProjectCacheProvider.java:122]: Somehow didn't notice

**Review by andrii-bodnar — APPROVED** [review]

**Pull request merged**